interface/reports/patient_edu_web_lookup.php

```diff
@@ -55,76 +44,51 @@ function searchResultsPopup(search_term,link) {
   }
 </script>
 
-<title><?php echo htmlspecialchars( xl('Find Patient Education Materials'), ENT_NOQUOTES); ?></title>
 </head>
 
-<body class="body_top">
-
-<span class='title'><?php echo htmlspecialchars( xl('Web Search'), ENT_NOQUOTES); ?> - <?php echo htmlspecialchars( xl('Patient Education Materials'), ENT_NOQUOTES); ?></span>
-
+<body class="body_top" onload="javascripts:document.forms[0].form_import_data.focus()">
+<div class="container">
+  <div class="row">
+<!-- -->
+    <div class='title'><?php echo  xlt('Web Search'); ?> - <?php echo xlt('Patient Education Materials'); ?></div>
+  </div>
+</div>
 <form method='post' action='patient_edu_web_lookup.php' id='theform' onsubmit='return top.restoreSession()'>
 
-<div id="report_parameters">
-
-    <table>
-        <tr class="misc-internet-search">
-  <td>
-	<div style='float:left'>
-	<table class='text'>
-		<tr>
-			<td>
-			   <?php echo htmlspecialchars( xl('Search in'), ENT_NOQUOTES);
-                echo '&nbsp;&nbsp;';
+    <div class="form-group">
+       <div class="row">
+			   <?php echo xlt('Search in'); 
+                             echo '&nbsp;&nbsp;';
 				echo "<select name='form_lookup_at'>\n";
 				foreach ($websites as $key => $value) {
-				  echo "    <option value='" . htmlspecialchars($key, ENT_QUOTES) . "'";
+				  echo "    <option value='" . xlt($key) . "'";
 				  if ($key == $form_lookup_at) echo ' selected';
-				  echo ">" . htmlspecialchars( xl($key), ENT_NOQUOTES) . "</option>\n";
+				  echo ">" .  xlt($key) . "</option>\n";
 				}
 				echo "</select>"; ?>
-			</td>
-        </tr>
-        <tr>
-            <td>
-			   <input type='text' name='form_diagnosis' size='60' value='<?php echo htmlspecialchars($form_diagnosis, ENT_QUOTES); ?>'
-				title='<?php echo htmlspecialchars( xl('Search Text'), ENT_QUOTES); ?>'>
-			</td>
-		</tr>
-	</table>
-
-	</div>
-
-  </td>
-  <td align='left' valign='middle' height="100%">
-	<table style='border-left:1px solid; width:100%; height:100%' >
-		<tr>
-			<td>
-				<div style='margin-left:15px'>
-					<a href='#' class='css_button' onclick='top.restoreSession(); $("#theform").submit();'>
-					<span>
-						<?php echo htmlspecialchars( xl('Submit'), ENT_NOQUOTES); ?>
-					</span>
-					</a>
-
-				</div>
-			</td>
-		</tr>
-	</table>
-  </td>
- </tr>
-</table>
 
-</div> <!-- end of parameters -->
+       </div>
+       <div class="row">
```

Comment:
This should be fixed in the searchHelpBox solution below.
Also in the form tag above, try the form-horizontal class:
``````
<form method='post' class='form-horizontal' action='patient_edu_web_lookup.php' id='theform' onsubmit='return top.restoreSession()'>